units/patch_intent.go

```diff
@@ -213,14 +220,14 @@ func (j *patchIntentProcessor) finishPatch(patchDoc *patch.Patch, githubOauthTok
 		return err
 	}
 
-	if j.Intent.ShouldFinalizePatch() {
-		if _, err := model.FinalizePatch(patchDoc, j.Intent.RequesterIdentity(), githubOauthToken); err != nil {
+	if j.intent.ShouldFinalizePatch() {
+		if _, err := model.FinalizePatch(patchDoc, j.intent.RequesterIdentity(), githubOauthToken); err != nil {
 			grip.Error(message.WrapError(err, message.Fields{
 				"message":     "Failed to finalize patch document",
 				"job":         j.ID(),
 				"patch_id":    j.PatchID,
-				"intent_type": j.Intent.GetType(),
-				"intent_id":   j.Intent.ID(),
+				"intent_type": j.IntentType,
+				"intent_id":   j.IntentID,
```

Comment:
While you're here, can you add "source":      "patch intents" to this logging statement?